package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "database",
+  "version": "1.0.0",
+  "description": "Backend",
+  "main": "index.js",
```

Comment:
Why is the **main** declared as **index.js** ?
@digi0ps

Reply:
It's a npm config thing, doesn't matter @TheMightyRaider